### PR TITLE
Deploy-361 - Readme changes

### PR DESCRIPTION
**Pull request by subaprasanna**
The readme changes in hello-world to get the charts from charts repo rather than charts-test

### CODE REVIEW

**Comment by melahn** [comment]
Why are two of the build steps failing?

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ helm install stable/nginx-ingress --namespace example
 3. Add the helm repository so that chart dependencies can be pulled:
 
 ```bash
-helm repo add alfresco-test https://alfresco.github.io/charts-test/incubator
+helm repo add alfresco-test https://alfresco.github.io/charts/incubator
```
Comment (rgauss) [inline README.md]: Actually, looking closer, I think we wanted to consistently name this repo alfresco-incubator everywhere, right?

**Comment by rgauss** [comment]
> Why are two of the build steps failing?

What failures are you seeing?

**Comment by melahn** [comment]
I was referring to these build failures which would not seem to have anything to do with the change but I was surprised they were red:
https://bamboo.alfresco.com/bamboo/browse/PS-MKDHW2-2 
https://bamboo.alfresco.com/bamboo/browse/PS-MKDHW2-3

**Comment by jottley** [comment]
Those failures are in Master correct?  That would be because needed structure is not there yet. The DEPLOY-310 branch is passing


On Feb 13, 2018, at 10:26 AM, Gregory Melahn <notifications@github.com<mailto:notifications@github.com>> wrote:


I was referring to these build failures which would not seem to have anything to do with the change but I was surprised they were red:
https://bamboo.alfresco.com/bamboo/browse/PS-MKDHW2-2
https://bamboo.alfresco.com/bamboo/browse/PS-MKDHW2-3

**Comment by rgauss** [comment]
> Those failures are in Master correct?  That would be because needed structure is not there yet. The DEPLOY-310 branch is passing

Those were on this DEPLOY-361 branch.

I believe the failures are just @svidrascu changing the Bamboo build, i.e. one task is expecting a version.properties that doesn't exist.

File: README.md (hunk 1 above)
Comment (melahn) [reply README.md]: yes, calling it alfresco-test seems inconsistent

Comment (jottley) [reply README.md]: Since Sergiu should be home I've updated the name.

Comment (nicdoye) [reply README.md]: Dudes - the URL is http://kubernetes-charts.alfresco.com/incubator not the github one. Sorry :-(

**Pull request closed without merging**